### PR TITLE
Adds Tarot to the Chaos Staff

### PR DESCRIPTION
**Pull request by datlo**
## What Does This PR Do

Adds three new effects to the chaos staff 
- Spawns and immediately use a random tarot card (Neutral effect)
- Spawns a tarot card pack in your backpack (Gift)
- Spawns a traitor-tier tarot deck generator in your backpack (Great gift)
## Why It's Good For The Game

yodawg

## Testing

it just works
~~Tweaked the probs to make the effects more likely to happen and checked that everything worked as intended~~
## Changelog
:cl:
add: Added new Tarot-related effects to the chaos staff.
/:cl:

### CODE REVIEW

**Comment by Sirryan2002** [comment]
Me with PRs that don't have a properly filled out testing section

giphy-ezgif com-webp-to-gif-converter

File: code/modules/projectiles/guns/chaos_bolt.dm [hunk 1]
```diff
@@ -301,12 +301,19 @@
 		if("bike horn")
 			item_to_summon = /obj/item/bikehorn
 			explosion_amount = rand(2, 3)
-
+		if("tarot card")
```
Comment (Sirryan2002) [inline code/modules/projectiles/guns/chaos_bolt.dm:304]:
```suggestion
		if("tarot card")
			#warn TEST ME!
```

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]
Does "everything works" mean that you were able to generate all added effects in-game without runtimes or bugs?

**Comment by datlo** [comment]
Thats correct yeah, all effects were triggered without runtime or bug

I'll do another round of tests with https://github.com/ParadiseSS13/Paradise/pull/25919 changes included to make sure there's no unexpected behaviour

**Review by Sirryan2002 — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]

**Pull request merged**